.github/workflows/ephemeral-instances-pr-comment.yml

```diff
@@ -26,6 +29,9 @@ jobs:
           fi
 
   handle-pull-request-event:
+    permissions:
+      contents: read
```

Comment:
```suggestion
    permissions:
      id-token: write
      contents: read
```

Reply:
i'm slow today - what's the `id-token` needed for? 😅

Reply:
I'm assuming for the "Generate a GitHub app installation token" step